.github/workflows/testadapters.yaml

```diff
@@ -22,6 +22,18 @@ on:
       patternfile_name:
         required: true
         type: string   
+      k8s_version:  
+        required: true
+        type: string   
+      sm_version:  
```

Comment:
How do we guarantee the the pattern file and these separate version inputs will be the same?

I assume the answer is that we are not. And that the individual configuring this workflow has to account for that.

Reply:
In the workflow that is used to call this one. Automatically the latest service mesh version is set ,both in patternfile as well as in parameter. No manual stuff

Reply:
That's great. This leads me to wonder why these parameters are being specified twice then.

Reply:
We have to pass the sm version so that we can reflect that onto the test results. If not for specifying twice, the other way would have been to extract directly from patternfile, which will not be a good idea considering we want this to be as general as possible.